File: ympa_fsharp_functions_server/src/model/OfferRecommendationDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.OfferForRecommendationDTO
open ympa_fsharp_functions_server.Model.OfferRecommendationInfoDTO

module OfferRecommendationDTO =

  //#region OfferRecommendationDTO

  [<CLIMutable>]
  type OfferRecommendationDTO = {
    [<JsonProperty(PropertyName = "offer")>]
    Offer : OfferForRecommendationDTO;
    [<JsonProperty(PropertyName = "recommendation")>]
    Recommendation : OfferRecommendationInfoDTO;
  }

  //#endregion

File: src/main/scala/org/openapitools/server/model/OfferRecommendationDTO.scala
package org.openapitools.server.model


/**
 * Информация о состоянии цен и рекомендации. 
 *
 * @param offer  for example: ''null''
 * @param recommendation  for example: ''null''
*/
final case class OfferRecommendationDTO (
  offer: Option[OfferForRecommendationDTO] = None,
  recommendation: Option[OfferRecommendationInfoDTO] = None
)

